src/main/java/org/example/Customer.java

```diff
@@ -3,11 +3,22 @@
 public class Customer {
 
     private final String username;
+    private int userid;
 
     public Customer(String username) {
```

Comment:
I'd remove this first constructor. We want all customers to be created with a userId.

Reply:
Done!